R/OntologyQueryFunctions.R

```diff
@@ -51,10 +53,15 @@ getOntoFromMeta <- function(mets, namesOrIds = "ids", includeRaMPids = FALSE, db
 
   rampid <- unique(df$rampId)
 
-  df2 <- db@api$getOntologiesForRampIDs(rampIds = rampid)
+  temp_ontologies_df <- getOntologies() %>%
```

Comment:
can you pass db to getOntologies here, otherwise it creates another db connection